### ISSUE TITLE
Support the rxvt terminal emulator

### ISSUE THREAD

**Issue by Dinduks** [ValveSoftware/steam-for-linux#3980]
Hi,

When installing Steam from the *.deb package, I got this error:

```
 steam-launcher depends on xterm | gnome-terminal | konsole; however:
  Package xterm is not installed.
  Package gnome-terminal is not installed.
  Package konsole is not installed.
```

This error is expected as I use none of the above, but *urxvt*. It would be great to support it.
My executable is `urxvt`, but other people might use `rxvt` or `rxvt-unicode`.

Thank you.

**Comment by Vinjul1704** [comment]
I agree, it would be great. I have gnome-terminal installed just because steam needs it, even if I use urxvt as my terminal.
